src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpCollectionExpressionRewriter.cs

```diff
@@ -325,10 +342,11 @@ SeparatedSyntaxList<CollectionElementSyntax> FixLeadingAndTrailingWhitespace(
         // Used to we can uniformly add the items correctly with the requested (but optional) indentation.  And so that
         // commas are added properly to the sequence.
         void CreateAndAddElements(
-            ImmutableArray<CollectionExpressionMatch<TMatchNode>> matches,
+            ImmutableArray<CollectionMatch<TMatchNode>> matches,
             ArrayBuilder<SyntaxNodeOrToken> nodesAndTokens,
             string? preferredIndentation,
-            bool forceTrailingComma)
+            bool forceTrailingComma,
+            bool moreToCome)
```

Comment:
>moreToCome [](http://example.com/codeflow?start=17&length=10)

nit: might be nice if this method and AddCommaIfMissing were consistent on using last or moreToCome